api/src/main/java/eu/darkbot/api/game/other/Locatable.java

```diff
@@ -76,6 +77,7 @@ default double angleTo(@NotNull Locatable other) {
     }
 
     //Locatable implementation
+    @ToString
     class LocatableImpl implements Locatable {
         private final double x;
         private final double y;
```

Comment:
just `@Value` it, will generate all the class: getters, fields as final, constructor, equals, hashcode, and tostring

Reply:
Current implmenetation if equals in LocatableImpl & LocationImpl check if is instance of Locatable interface. Lombok will check if is instance of implmentation class